### ISSUE TITLE
Add first implementation for using OrientDB as a DB engine

### ISSUE THREAD

**Issue by f-lombardo** [smeup/jariko#47]
In addition to SQL DBs, we would like to support NoSQL DBs to as persistent engines for our project.
One of the first NoSQL DBs we would like to support is OrientDB.
As a first step in this implementation we wrote a skeleton test for this issue: `DBOrientTest`.
This test should:
1.  start an embedded instance of OrientDB (as we do with HSQLDB, see `DBSQLTest`);
2. create an OrientDB class;
3. check that our implementation can read the schema of this class.

**Comment by f-lombardo** [comment]
@ftomassetti , 
I think you closed this issue erroneously, isn't it?

**Comment by ftomassetti** [comment]
Yes, the reason for this is that the commit indicated as the description "closes #47" and GitHub therefore marked this issue as closed. Now, I wrote that comment because we have an issue 47 in our own copy of the repo, which is different. I will discuss with @mctaverna to understand how we can avoid similar issues in the future. Sorry for the troubles
